website/search/views.py

```diff
@@ -239,17 +241,31 @@ def search_share_stats():
 
 @handle_search_errors
 def search_share_atom(**kwargs):
-    q = request.args.get('q', '*')
-    sort = request.args.get('sort', 'dateUpdated')
-
-    # we want the results per page to be constant between pages
-    # TODO -  move this functionality into build_query in util
+    json_query = request.args.get('jsonQuery')
```

Comment:
The query is an elasticsearch DSL json blob, and it is a non-trivial effort to write something that can dump and parse it well (it can be arbitrarily deep).